docs/developer.md

```diff
@@ -1,218 +1,110 @@
-Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}
-```
 
-#### 4 Validate Java is Installed Correctly
 
-```bash
-$ java -version
-```
+### Verify System Environment
 
-This should produce output similar to the following:
+Check and verify that your environment is properly configured with the following software for building and installing the operator:
 
-```bash
-java version "11.0.1" 2018-10-16 LTS
-...
-```
+| Software | Verify | Expected Output|
+| ---------| ----------------------------|----------------|
+| Docker   | `$ docker run hello world`  | `Hello from Docker!`|
+| Kubernetes | `$ kubectl version` | `Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-04T04:49:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"darwin/amd64"}<br>Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.11", GitCommit:"637c7e288581ee40ab4ca210618a89a555b6e7e9", GitTreeState:"clean", BuildDate:"2018-11-26T14:25:46Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}` |
+| Helm | `$ helm version` | `Client: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}<br>Server: &version.Version{SemVer:"v2.12.3", GitCommit:"eecf22f77df5f65c823aacd2dbd30ae6c65f186e", GitTreeState:"clean"}` |
+| Java | `java version` | `java version "11.0.1" 2018-10-16 LTS` |
+| Maven | `mvn version` | `Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T14:33:14-04:00) <br> Maven home: /Users/username/Downloads/apache-maven-3.5.4 <br> Java version: 11.0.1, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.1.jdk/Contents/Home` |
 
-#### 5 Validate Maven is Installed Correctly
+## Build the Operator
 
-```bash
-$ mvn -version
-```
+To build the operator without running any tests, do the following:
 
-This should produce output similar to the following:
-
-```bash
-Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T14:33:14-04:00)
-Maven home: /Users/username/Downloads/apache-maven-3.5.4
-Java version: 11.0.1, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.1.jdk/Contents/Home
-...
-```
-
-## How to Build the Operator, Without Running Any Tests
-
-1. Clone and check out the `1.0` branch of the [GitHub repository](https://github.com/oracle/coherence-operator).
-
-   If you do not have a maven `settings.xml` file, create one.  If you
-   have one, make sure the following is included in your `default` profile.
-   All of the maven commands in this document are assumed to use this
-   `settings.xml` file.
+1. Clone and check out the operator version `1.0` from the [GitHub repository](https://github.com/oracle/coherence-operator).
```

Comment:
Should we specifically mention version 1.0 or the current version?

Reply:
we can say current version for now. I can make that change.